crates/astria-core/src/primitive/v1/mod.rs

```diff
@@ -16,6 +19,14 @@ use crate::{
 };
 
 pub const ADDRESS_LEN: usize = 20;
+/// The human readable prefix of astria addresses (also known as bech32 HRP).
+pub const HUMAN_READABLE_ADDRESS_PREFIX: &str = "astria";
```

Comment:
for cosmos chains, this is configurable - would there be an easy way to configure this prefix based on genesis info for instance?

Reply:
Tough to do right now, punting.